frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/services/ReverificationResultService.java

```diff
@@ -88,17 +88,25 @@ public TokenResponse sendTokenRequest(TokenRequest tokenRequest) {
             int count = 0;
             int maxTries = 2;
             TokenResponse tokenResponse;
+
             do {
-                if (count > 0) LOG.warn("Retrying IPV access token request");
+                if (count > 0) {
+                    LOG.warn("Retrying IPV access token request");
+                }
+
                 count++;
                 var httpRequest = tokenRequest.toHTTPRequest();
                 var httpResponse = httpRequest.send();
+
                 tokenResponse = TokenResponse.parse(httpResponse);
+
                 if (!tokenResponse.indicatesSuccess()) {
-                    HTTPResponse response = tokenResponse.toHTTPResponse();
+                    var response = tokenResponse.toErrorResponse();
                     LOG.warn(
-                            "Unsuccessful %s response from IPV token endpoint on attempt %d: %s ",
-                            response.getStatusCode(), count, response.getContent());
+                            "Unsuccessful {} response from IPV token endpoint on attempt: {}; error: {}",
+                            response.toHTTPResponse().getStatusCode(),
+                            count,
+                            httpResponse.getContent());
```

Comment:
Agree we can't guarantee IPV responses don't contain PII but IPV is an internal system so we should have a high level of confidence it won't.  Also the intention of the error description is that it will provide additional information to the error code so I think we are safe to log it and it should make triage easier.